src/App.test.jsx

```diff
@@ -10,6 +15,39 @@ test("renders without error", () => {
   expect(appComponent.length).toBe(1);
 });
 
+test("renders FAQ page", () => {
+  const wrapper = shallow(<FAQ />);
+  const faqComponent = findByTestAttr(wrapper, "FAQ-page");
+  expect(faqComponent.length).toBe(1);
+});
+
+test("renders Information page", () => {
+  const wrapper = shallow(<Information />);
+  const infoComponent = findByTestAttr(wrapper, "information-display");
+  expect(infoComponent.length).toBe(1);
+});
+
+test("renders Profile page", () => {
+  const wrapper = shallow(<Profile />);
+  const profileComponent = findByTestAttr(wrapper, "Profile-page");
+  expect(profileComponent.length).toBe(1);
+});
+
+test("renders Registration page", () => {
+  const wrapper = shallow(<Registration />);
+  const registrationComponent = findByTestAttr(wrapper, "Registration-page");
+  expect(registrationComponent.length).toBe(1);
+});
+
+test("renders Accordion List", () => {
+  const wrapper = shallow(<AccordionList />);
+  const accordionListComponent = findByTestAttr(
+    wrapper,
+    "component-accordionList"
+  );
+  expect(accordionListComponent.length).toBe(1);
+});
+
```

Comment:
:wrench: Rebase this branch and remove these lines.

Reply:
Addressed:  a379eb7b3721a74ff9e1e84d867dfc18c86eb75b